.circleci/test.sh

```diff
@@ -1,21 +1,29 @@
 #!/bin/bash
 
+# override CircleCi's default run settings
+set +e
+set +o pipefail
```

Comment:
I don't like CircleCI's new defaults where a given job stop executing after the first test failure. I prefer running all the tests no matter what, to see if a given commit makes multiple tests fail. I hope you agree with me.